Composer/packages/client/src/store/persistence/FilePersistence.ts

```diff
@@ -28,6 +29,14 @@ class FilePersistence {
   private _files: { [fileName: string]: FileOperation };
```

Comment:
fileOperators?

Reply:
also, i don't think you should maintain a operator per file, operator should be stateless, ie, static functions.